css/style.css

```diff
@@ -0,0 +1,2921 @@
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Montserrat Regular'), local('Montserrat-Regular'),
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff2') format('woff2'), 
+  url('../fonts/montserrat-v14-latin_cyrillic-regular.woff') format('woff'); 
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 500;
+  src: local('Montserrat Medium'), local('Montserrat-Medium'),
+  url('../fonts/montserrat-v14-latin_cyrillic-500.woff2') format('woff2'),
+  url('../fonts/montserrat-v14-latin_cyrillic-500.woff') format('woff'); 
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 600;
+  src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'),
+  url('../fonts/montserrat-v14-latin_cyrillic-600.woff2') format('woff2'),
+  url('../fonts/montserrat-v14-latin_cyrillic-600.woff') format('woff');
+}
+
+@font-face {
+  font-family: 'Montserrat';
+  font-style: normal;
+  font-weight: 700;
+  src: local('Montserrat Bold'), local('Montserrat-Bold'),
+  url('../fonts/montserrat-v14-latin_cyrillic-700.woff2') format('woff2'),
+  url('../fonts/montserrat-v14-latin_cyrillic-700.woff') format('woff'); 
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 1.2;
+  font-weight: normal;
+  color: #000; 
+  margin: 0;
+}
+
+.container {
+  max-width: 1440px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+  margin-right: 15px;
+}
+
+.header__logo {
+  display: inline-flex;
+  background-color: #fff;
+  padding-left: 42px;
+}
+
+.header__logo-img {
+  background-color: #fff;
+}
+
+.menu {
+  margin-left: 73px;
+}
+
+.menu__link {
+  font-weight: 600;
+  font-size: 15px;
+  line-height: 18px;
+  padding: 6px;
+  margin-right: 40px;
+  color: #000000;
```

Comment:
Codacy found an issue: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)